components/ai_chat/resources/page/components/conversation_list/index.tsx

```diff
@@ -28,15 +29,22 @@ function ConversationList() {
     isGenerating,
     conversationHistory,
     suggestedQuestions,
-    shouldDisableUserInput
+    shouldDisableUserInput,
+    hasAcceptedAgreement,
+    siteInfo,
+    shouldSendPageContents
   } = context
 
   const portalRefs = React.useRef<Map<number, Element>>(new Map())
 
-  const showSuggestions: boolean =
+  let showSuggestions: boolean =
     suggestedQuestions.length > 0 ||
     SUGGESTION_STATUS_SHOW_BUTTON.includes(context.suggestionStatus)
 
+  if (hasAcceptedAgreement) {
+    showSuggestions = false
+  }
+
```

Comment:
Why are we hiding suggestions when the user is opted in? I don't see any suggestions anymore.

Reply:
mistake, fixed